src/utils.js

```diff
@@ -0,0 +1,249 @@
+    await turingHelper.initialize();
+
+    const mangataHelper = new MangataHelper(MangataRococo);
+    await mangataHelper.initialize();
+
+    const actionSelected = await select({
+        message: 'Select an action to perform',
+        choices: actions,
+    });
+
+    switch (actionSelected) {
+    case 'examine-mangata-pools': {
+        const pools = await mangataHelper.getPools({ isPromoted: true, thousandSeparator: true });
+        console.log(pools);
+
+        break;
+    }
+    case 'get-mangata-assets':
+        await mangataHelper.getAssets();
+        break;
+    case 'transfer-tur-on-mangata':
+    {
+        const seed = 'visit hazard toss claim edge century bunker correct great very insect calm';
+        const account = keyring.addUri(seed, undefined);
+        console.log('account', account);
+        const token = _.find(Mangata.assets, { symbol: 'TUR' });
+        console.log('token', token);
+        await mangataHelper.transferToken({
+            keyPair: account.pair, tokenId: token.id, decimals: token.decimals, dest: '5HDjVvqaSMQ8YuFVo9yghvqUWLpW5vZ5s5EagUiiYF16ikc3', amount: 100,
+        });
+        break;
+    }
+    case 'transfer-mgx-on-mangata':
+    {
+        const seed = 'visit hazard toss claim edge century bunker correct great very insect calm';
+        const json = keyring.addFromMnemonic(seed, undefined);
+
+        const source = new Account(json);
+        await source.init();
+        source.print();
+
+        const mgxToken = source.getAssetByChainAndSymbol('mangata', 'MGX');
+        // console.log('mgxToken', mgxToken);
+
+        if (mgxToken.balance <= 0) {
+            throw new Error(`Not enough MGX balance, current: ${mgxToken.balance} ...`);
+        }
+
+        const token = _.find(Mangata.assets, { symbol: 'MGX' });
+        // const destAddress = dest.getAddressByChain('mangata');
+
+        // console.log('destAddress', destAddress);
+
+        await mangataHelper.transferToken({
+            keyPair: source.pair, tokenId: token.id, decimals: token.decimals, dest: '5Hg97z8iVvkZiViemwp5gfBC4SPDBmGx3qJHr7URbPdtZQka', amount: 100,
+        });
+
+        break;
+    }
+    case 'transfer-tur-on-turing': {
+        const chain = 'turing';
+        const symbol = 'TUR';
+
+        const source = new Account();
+        await source.init();
+        source.print();
+
+        const dest = new Account(3);
+        await dest.init();
+        dest.print();
+
+        const asset = source.getAssetByChainAndSymbol(chain, symbol);
+        console.log('asset', asset);
+
+        if (asset.balance <= 0) {
+            throw new Error(`Not enough ${symbol} balance ...`);
+        }
+
+        const destAddress = dest.getAddressByChain('turing');
+
+        console.log('destAddress', destAddress);
+
+        await turingHelper.transferTUR({
+            recipient: destAddress, amount: 50, keyPair: source.pair,
+        });
+
+        console.log('Transfer completed!');
+        break;
+    }
```

Comment:
All these functions are not used yet, but will be very handy for Mangata on which there’s no other easy way to read balance or transfer tokens.